package.json

```diff
@@ -1,16 +1,29 @@
 {
   "name": "react-financial-portal",
-  "version": "0.1.0",
   "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint .",
+    "preview": "vite preview"
+  },
```

Comment:
**suggestion**: Consider using the latest stable versions of dependencies to ensure compatibility and access to the latest features.